content/docs/command-reference/repro.md

```diff
@@ -110,7 +110,8 @@ up-to-date and only execute the final stage.
   reproduced independently.
 
 - `-R`, `--recursive` - determines the stages to reproduce by searching each
-  target directory (if any) and their subdirectories.
+  directory and their subdirectories. These directories can be given as
+  `targets`.
```

Comment:
@jorgeorpinel does the option description need to explicitly convey that -R will work when at least one of the target is directory.

Reply:
> searching each directory and their subdirectories

This is not clear. What directory? "target directory" was better, (even if the next sentence explains it — it should be clear first that -R expands what `targets` accepts).
Also "directory and their" mixes singular and plural (incorrect grammar).

> need to explicitly convey that -R will work when at least one of the target is directory

Yes, good idea. Let's be explicit about that. Use the phrase "only has an effect if" please.

Reply:
Also, you may have missed

> Make sure options -c and -P are also clear and complete, as they also allow to target directories in different ways.

From #1861

Reply:
`-c` and `-P` option are complete I think.
Like `-P` doesn't require `target` (It is working without them).
-c` option's description also looks completes.

Reply:
Looks like -c is OK but -P is not complete. I'll reopen this review... Resolving here.